src/api/helpers.js

```diff
@@ -0,0 +1,203 @@
+    "Batman: Arkham City",
+    "The Witness",
+    "Journey",
+    "Uncharted 2: Among Thieves",
+    "Overwatch",
+    "Deus Ex",
+    "Baldur's Gate II: Shadows of Amn",
+    "Ms. Pac-Man",
+    "Counter-Strike 1.6",
+    "Left 4 Dead 2",
+    "EarthBound",
+    "Resident Evil",
+    "Diablo II",
+    "StarCraft",
+    "World of Warcraft",
+    "Star Wars: Knights of the Old Republic",
+    "Fallout: New Vegas",
+    "Final Fantasy VI",
+    "Mass Effect 2",
+    "Pokémon Yellow",
+    "Bloodborne",
+    "Metroid Prime",
+    "Resident Evil 4",
+    "Shadow of the Colossus",
+    "Metal Gear Solid",
+    "God of War",
+    "The Witcher 3: Wild Hunt",
+    "BioShock",
+    "Sid Meier's Civilization IV",
+    "The Legend of Zelda: Ocarina of Time",
+    "Minecraft",
+    "Halo: Combat Evolved",
+    "Half-Life",
+    "Metal Gear Solid 3: Snake Eater",
+    "The Last of Us",
+    "Doom",
+    "Chrono Trigger",
+    "Portal",
+    "Dark Souls",
+    "Street Fighter II",
+    "Super Mario Bros.",
+    "Halo 2",
+    "Castlevania: Symphony of the Night",
+    "Grand Theft Auto V",
+    "Super Mario 64",
+    "Red Dead Redemption",
+    "Half-Life 2",
+    "Tetris",
+    "Super Mario Bros. 3",
+    "The Legend of Zelda: Breath of the Wild",
+    "Super Metroid",
+    "Portal 2",
+    "The Legend of Zelda: A Link to the Past",
+    "Super Mario World",
+  ],
+}
+
+let category = ["xbox", "ps", "pc"]
+function fillCategoty(arr) {
+  let random = Math.floor(Math.random() * 4)
+  if (random === 3) {
+    return []
+  }
+  return [arr[random]]
+}
+
+function randomInteger(min, max) {
+  let rand = min + Math.random() * (max + 1 - min)
+  return Math.floor(rand)
+}
+
+function fillSrc(arr) {
+  let random = Math.floor(Math.random() * arr.length)
+  return imgSrc[random]
+}
+
+const imgSrc = [
+  "https://i.pinimg.com/564x/3b/f7/c6/3bf7c6a4d52505ede9e3582480d62545.jpg",
+  "https://i.pinimg.com/564x/9a/83/71/9a83710f141c4977af1b4630546f2f1d.jpg",
+  "https://i.pinimg.com/564x/a2/60/3a/a2603a2e81105cb1f6b5e39f77fdf953.jpg",
+  "https://i.pinimg.com/564x/16/cd/2b/16cd2b9754498971e80a53c3925c7c97.jpg",
+  "https://i.pinimg.com/564x/4e/e4/b4/4ee4b45963b6e6b84b8c1a02eebb7fdd.jpg",
+  "https://i.pinimg.com/564x/ad/ba/a4/adbaa41b81502bad7de3c24727d19560.jpg",
+]
+
+for (let i = 4; i < 105; i++) {
+  let b = {
+    id: i,
+    category: fillCategoty(category),
+    title: "",
+    imgSrc: fillSrc(imgSrc),
+    price: `$${randomInteger(15, 50)}.${randomInteger(10, 99)}`,
+    description:
+      "Mauris suscipit dui eu elit pellentesque ornare. Etiam arcu quam, pulvinar ac lorem eget, placerat semper arcu. Suspendisse maximus dui non convallis faucibus. Nam enim erat, aliquam at iaculis consequat, consequat eget purus.",
+    ageAllowed: `${randomInteger(8, 18)}+`,
+    releaseDate: randomInteger(1350000000000, 1400000000000),
+  }
+  b.title = titles.title[i]
+  dataG.push(b)
+}
+
+export const gamesData = () => writeFile("games.json", dataG)
```

Comment:
Why do you need this code for pasting games into json file? It should be done once and never reused again